src/libraries/System.Net.Ping/src/System/Net/NetworkInformation/Ping.cs

```diff
@@ -208,6 +358,34 @@ public PingReply Send(string hostNameOrAddress, int timeout, byte[] buffer, Ping
             return GetAddressAndSend(hostNameOrAddress, timeout, buffer, options);
         }
 
+        /// <summary>
+        /// Attempts to send an Internet Control Message Protocol (ICMP) echo message to the computer that has the specified <see cref="IPAddress"/>,
+        /// and receive a corresponding ICMP echo reply message from that computer.
+        /// </summary>
+        /// <param name="address">
+        /// An <see cref="IPAddress"/> that identifies the computer that is the destination for the ICMP echo message.
+        /// </param>
+        /// <param name="timeout">
+        /// An <see cref="int"/> value that specifies the maximum number of milliseconds (after sending the echo message) to wait for the ICMP echo reply message.
+        /// </param>
+        /// <param name="buffer">
+        /// A <see cref="byte"/> array that contains data to be sent with the ICMP echo message and returned in the ICMP echo reply message.
+        /// The array cannot contain more than 65,500 bytes.
```

Comment:
here you use 65,500, but the ArgumentException doc comment uses 65500 so there is some inconsistency (same for other overloads).

Reply:
Fixed, thanks